src/modules/out_jnibuf.c

```diff
@@ -0,0 +1,157 @@
+// Dummy output.
+//
+// Copyright (c) 2021, fridtjof <fridtjof@das-labor.org>
+//
+// Permission to use, copy, modify, and/or distribute this software for any
+// purpose with or without fee is hereby granted, provided that the above
+// copyright notice and this permission notice appear in all copies.
+//
+// THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+// WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+// ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+// WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+// ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+// OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+#include "../os/sh_tty_sled_JniSled.h"
+
+#include <types.h>
+#include <timers.h>
+#include <assert.h>
+#include <string.h>
+#include <stdlib.h>
+#include "stdio.h"
+
+#define PPOS(x, y) (x + (y * matx))
+
+static size_t JNI_BUFFER_SIZE;
+static jintArray buffers[2];
+static jint* working_buffer;
+static jint* currentBufferPtr;
+static int currentBuffer = -1;
+#define NEXTBUFFER ((currentBuffer + 1) % 2)
+
+extern JNIEnv *pJniEnv;
+#define JniEnv (*pJniEnv)
+extern jobject jo_jni_sled;
+
+static jclass jc_jni_sled;
+static jmethodID jm_setCurrentBuffer;
+static jfieldID jf_matx;
+static jfieldID jf_maty;
+
+static int matx;
+static int maty;
+
+void ignoreException() {
```

Comment:
this should probably called something along the lines of `print_exception` rather than ignore exception..

Reply:
resolved in b88d4e1cf56e84a5b75b55709178a3a8626f5472